File: ContosoWebFormsScaffolding/Samples.VB/Models/DatabaseInitializer.vb
Imports System.Data.Entity

Public Class DatabaseInitializer
    Inherits DropCreateDatabaseAlways(Of ApplicationDbContext)
    Protected Overrides Sub Seed(context As ApplicationDbContext)
        Dim categories = New List(Of Category)() From { _
            New Category() With { _
                .Name = "Beverages" _
            }, _
            New Category() With { _
                .Name = "Fruit" _
            } _
        }

        categories.ForEach(Function(c) context.Categories.Add(c))

    End Sub
End Class

File: extensibilty-for-asp.net-devs/ContosoWebFormsScaffolding/Samples.CS/Models/DatabaseInitializer.cs
using Samples.Associations;
using System;
using System.Collections.Generic;
using System.Data.Entity;
using System.Linq;
using System.Web;

namespace Samples.Models
{
    public class DatabaseInitializer : DropCreateDatabaseIfModelChanges<ApplicationDbContext>
    {
        protected override void Seed(ApplicationDbContext context)
        {
            var categories = new List<Category> {
                new Category {Name="Beverages"},
                new Category {Name="Fruit"}
            };

            categories.ForEach(c => context.Categories.Add(c));

        }
    }
}